tox.ini

```diff
@@ -93,3 +93,13 @@ deps =
     -r{toxinidir}/requirements.txt
 commands =
     pytest -v --tb native {[vars]tst_path}integration/relation_tests --log-cli-level=INFO -s {posargs} --durations=0 --asyncio-mode=auto
+
+[testenv:integration]
+description = Run all integration tests
+deps =
+    pytest
+    juju
+    pytest-operator
+    -r{toxinidir}/requirements.txt
+commands =
+    pytest -v --tb native --ignore={[vars]tst_path}unit --log-cli-level=INFO -s {posargs} --durations=0 --asyncio-mode=auto
```

Comment:
Paulo had a good suggestion about movin the asyncio mode to `pyproject.toml`, so it is not needed to specify it in each environment that we have in `tox.ini`.

```toml
[tool.pytest.ini_options]
minversion = "6.0"
log_cli_level = "INFO"
asyncio_mode = "auto"
```

This is optional.

Reply:
great idea! I like this :)